### ISSUE TITLE
[Feature] There seems to be no bindings for "ConstantArray"

### ISSUE THREAD

**Issue by wavpro** [ApsarasX/llvm-bindings#17]
**Which LLVM APIs do you want llvm-bindings to provide**
There seems to be no bindings for "ConstantArray", which is quite an important datatype since it's needed for most string implementations. Could it be added to the bindings or is there some issue preventing that?

**Comment by ApsarasX** [comment]
OK, I will add `class ConstantArray` as soon as possible.

**Comment by wavpro** [comment]
Thank you so much!

**Comment by ApsarasX** [comment]
How are you going to use `class ConstantArray`?

I have added `class ConstantArray` in https://github.com/ApsarasX/llvm-bindings/commit/3b609cb2f1d98b5fd92ab3392d193363f6a1a2d7, but haven't pushed new version to npm yet.

In the commit, I just added two methods for `class ConstantArray`, which are:
```ts
public static get(type: ArrayType, values: Constant[]): Constant;

public getType(): ArrayType;
```

Do you need to use other methods of `class ConstantArray`?

**Comment by ApsarasX** [comment]
I have released v0.3.8 into npm.

For details, please see https://github.com/ApsarasX/llvm-bindings/releases/tag/v0.3.8

**Comment by dvoytenko** [comment]
@ApsarasX I think there might be a bug in https://github.com/ApsarasX/llvm-bindings/blob/c30e243bb96d2a891a672b0ffd1d1f7cdac6b9be/src/IR/Constants.cpp#L341

Shouldn't it be `ArrayType::IsClassOf`? See https://github.com/ApsarasX/llvm-bindings/pull/19.
